resources/public/api/conf/P2.0/application.raml

```diff
@@ -22,4 +22,16 @@ uses:
   sec: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/securitySchemes.raml
   headers: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/headers.raml
   annotations: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/annotations.raml
-  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
\ No newline at end of file
+  types: https://developer.service.hmrc.gov.uk/api-documentation/assets/common/modules/types.raml
+
```

Comment:
needs individuals/employments at base level
Check :- https://github.com/hmrc/individuals-details-api/blob/master/resources/public/api/conf/P1.0/application.raml
for example